### DISCUSSION TITLE
Nextjs 13.4 SWC plugin not working with typewind

### DISCUSSION THREAD

**Discussion by pawanrana1992** [Mokshit06/typewind#49] [General]
I am trying to use Typewind in nextjs 13.4.x with SWC support, but it is not working as per the suggested configuration for swc from Typewind side. I am getting error like this. can anyone please suggest what is the right configuration for Nextjs 13.4. I love this plugin and any how I need to use it. please help.
<img width="1160" alt="Screenshot 2566-07-07 at 1 03 56 PM" src="https://github.com/Mokshit06/typewind/assets/41635595/96065668-091a-4e33-9f46-bab97b3f1b02">
<img width="809" alt="Screenshot 2566-07-07 at 1 04 16 PM" src="https://github.com/Mokshit06/typewind/assets/41635595/f88690f6-6b3a-406e-ad80-4d08a97e31a7">
<img width="1380" alt="Screenshot 2566-07-07 at 1 05 19 PM" src="https://github.com/Mokshit06/typewind/assets/41635595/591a4e8f-8ad6-4c42-8d55-d964ba2e8833">

**Comment by Mokshit06** [comment]
This looks to be some issue with how SWC is calling the plugin, though I'm not entirely sure about it. Will look into the cause, you can use the babel plugin in the meantime
